### PR TITLE
Add font-face update documentation

### PR DESCRIPTION
**Pull request by thisisdano**
Adds documentation for the new setting and information related to https://github.com/uswds/uswds/pull/5108 — which changes our default `@font-face` output to include only `woff2` fonts unless `$theme-font-browser-compatibility: true` 

Note that I removed the `data/settings.yml` file since it was redundant with the contents of `data/settings`.

I didn't see any other references to the `@font-face` output outside of the Settings page. But it's worth double checking!

[Settings page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/dw-font-output-docs/documentation/settings/) →

### CODE REVIEW

**Comment by thisisdano** [comment]
Note that this will fail in Circle because of links to the `3.4.0` release which does not yet exist.

File: _data/settings/typography.yml [hunk 1]
```diff
@@ -36,6 +36,11 @@ contents:
     var: $theme-font-path
     default: '"../fonts"'
     type: path
+  - name: Browser compatibility mode
+    description: Output additional font formats (`woff` and `ttf`) to support IE11.
+    var: $theme-font-browser-compatibility
+    default: false
+    type: bool
```
Comment (mejiaj) [inline _data/settings/typography.yml:43]: [PR Preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/dw-font-output-docs/documentation/settings/#:~:text=false-,bool,-Output%20additional%20font) mentions `bool`, while others are `boolean`

image


```suggestion
    type: boolean
```

**Review by mejiaj — CHANGES_REQUESTED** [review]
Minor comment for consistency in how we refer to booleans in settings table.

Also, should we be including this information in [components/typography](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/dw-font-output-docs/components/typography/) too?

**Review by mejiaj — APPROVED** [review]
Updated component changelog to point to PR on USWDS in 92d294b031bbe6bf54000596abb707120893512d.

**Pull request merged**